lib/vintage_net_wizard/ap_mode.ex

```diff
@@ -64,7 +64,8 @@ defmodule VintageNetWizard.APMode do
 
   defp get_hostname() do
     {:ok, hostname} = :inet.gethostname()
-    to_string(hostname)
+    hostname_string = to_string(hostname)
+    Application.get_env(:vintage_net_wizard, :ssid, hostname_string)
```

Comment:
Do we need to validate this a bit more to ensure the value in `:ssid` is a valid SSID string?

Reply:
`vintage_net_wifi` will validate it (I just noticed that it doesn't validate ssids currently, but it will soon). I'm not sure that adding code here will make ssid-too-long mistakes easier to debug, so I feel ambivalent on adding it.

Reply:
I guess I'm more thinking about if someone does `ssid: :im_an_atom` or `ssid: 'wat is this chartist'` - Maybe our step is just to force `to_string` on this result, and let `vintage_net_wifi` do any other specific validations?

Reply:
I just PR'd `vintage_net_wifi` with a check for non-strings. It was already raising for non-strings, but the reason was incorrect. I'm leaning towards letting `vintage_net_wifi` do all of the checks until we find out that what it raises is hard to debug. That way if `vintage_net_wifi` changes the validation, we don't have to change it here as well.

Stylistically, I'd probably write the code like this:

```elixir
defp get_hostname() do
  case Application.get_env(:vintage_net_wizard, :ssid) do
    nil ->
      {:ok, hostname} = :inet.gethostname()
      to_string(hostname)
    ssid -> ssid
end
```

I don't have strong opinions on this, so I'm going to mark this as approved and when there's a second approve, I'll merge.